CHANGES.rst

```diff
@@ -8,48 +8,178 @@ https://hyperspy.readthedocs.io/en/latest/user_guide/changes.html
 
 .. towncrier-draft-entries:: |release| [UNRELEASED]
 
+.. towncrier release notes start
+
+.. _changes_2.0:
+
+2.0rc0 (2023-11-10)
+===================
+
+Release Highlights
+------------------
+- Hyperspy has split off some of the reading/writing files and domain specific functionalities into separate libraries!
+  
+  - `RosettaSciIO <https://hyperspy.org/rosettasciio>`_: A library for reading and writing scientific data files.
+    See `RosettaSciIO release notes <https://hyperspy.org/rosettasciio/changes.html>`_ for new features.
+  - `exSpy <https://exspy.readthedocs.io>`_: A library for EELS and EDS analysis.
+    See `exSpy release notes <https://hyperspy.org/exspy/changes.html>`_ for new features.
+  - `HoloSpy <https://holospy.readthedocs.io>`_: A library for analysis of (off-axis) electron holography data.
+    See `HoloSpy release notes <https://holospy.readthedocs.io/en/latest/changes.html>`_ for new features.
+
+- The :py:mod:`~.api.plot.markers` API has been refactored
+  - Lazy markers are now supported
```

Comment:
Formatting of the sublist is currently not working, might need blank line before and after as above?